Dockerfile

```diff
@@ -315,49 +215,8 @@ RUN /home/dark/install-targz-file \
 ############################
 # Yugabyte
 ############################
-COPY <<-"EOF" /home/dark/install-yugabyte
-#!/bin/bash
-
-# Script to install yugabyte
-
-set -e;
-case ${TARGETARCH} in
-  arm64)
-  URL=https://downloads.yugabyte.com/releases/2.20.0.0/yugabyte-2.20.0.0-b76-el8-aarch64.tar.gz
-  ;;
-  amd64)
-  URL=https://downloads.yugabyte.com/releases/2.20.0.0/yugabyte-2.20.0.0-b76-linux-x86_64.tar.gz
-  ;;
-  *) exit 1 ;;
-esac
-FILENAME=$(basename $URL)
-DIR=/home/dark/yugabyte
-mkdir -p $DIR
-cd $DIR
-curl -sSL -o $FILENAME $URL
-tar xvf $FILENAME --strip-components=1
-rm $FILENAME
-./bin/post_install.sh
-case $(dpkg --print-architecture) in
-  arm64)
-  sudo ln -sf /usr/lib/aarch64-linux-gnu/liblber-2.5.so.0 /usr/lib/aarch64-linux-gnu/liblber-2.4.so.2
-  sudo ln -sf /usr/lib/aarch64-linux-gnu/libldap-2.5.so.0 /usr/lib/aarch64-linux-gnu/libldap_r-2.4.so.2
-  ;;
-  amd64)
-  sudo ln -sf /usr/lib/x86_64-linux-gnu/liblber-2.5.so.0 /usr/lib/x86_64-linux-gnu/liblber-2.4.so.2
-  sudo ln -sf /usr/lib/x86_64-linux-gnu/libldap-2.5.so.0 /usr/lib/x86_64-linux-gnu/libldap_r-2.4.so.2
-  ;;
-  *) exit 1;;
-esac
-# python on our system is called python3
-sed -i 's|#!/usr/bin/env python|#!/usr/bin/env python3|' ./bin/yugabyted
-cd ..
-EOF
-
-RUN sudo dos2unix /home/dark/install-yugabyte
-
-RUN sudo chown dark:dark /home/dark/install-yugabyte
-RUN chmod +x /home/dark/install-yugabyte
+ENV YUGABYTE_VERSION=2.20.0.0 \
```

Comment:
Could we do these as command line arguments, same as they're passed to instal-targz-file?